attributes/vc14.rb

```diff
@@ -62,11 +62,11 @@
 default['vcruntime']['vc14']['x64']['14.14.26429.4']['name']      = 'Microsoft Visual C++ 2017 Redistributable (x64) - 14.14.26429'
 
 # Microsoft Visual C++ 2019 Redistributable
```

Comment:
The intent is to leave all prior versions if someone wants to use that build but default to the latest version in the cookbook.